docs/src/components/Navbar/Links.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Hide, TextLink, Stack, Button, Popover } from "@kiwicom/orbit-components";
+import { MenuHamburger } from "@kiwicom/orbit-components/lib/icons";
+
+import PuzzleIcon from "../icons/Puzzle";
+import ColorBrushIcon from "../../images/streamline-light/color-brush.svg";
+import ReadArt from "../../images/streamline-light/read-art.svg";
+import IrisScanIcon from "../../images/streamline-light/iris-scan.svg";
+import ScriptIcon from "../../images/streamline-light/script.svg";
+
+type Link = "Components" | "Guides" | "Tokens" | "Accessibility" | "Foundation";
+
+interface Props {
+  links?: Link[];
+}
+
+const LinkIcon = ({
+  name,
+  isFirst,
+  isDesktop,
+}: {
+  name?: string;
+  isFirst?: boolean;
+  isDesktop?: boolean;
+}) => {
+  if (!isDesktop) {
+    switch (name) {
+      case "Components":
+        return <PuzzleIcon stroke="#252A31" width="20px" height="20px" />;
```

Comment:
Those styles (stroke, width, height) are common to all icons, so I'd rather have them stored in a const object and spread here, so that if one changes, they all change at the same time.